QuickBytes/src/main/java/com/quickbytes/model/Vendor.java

```diff
@@ -1,17 +1,47 @@
-package com.quickbytes.model;
+package com.springboot.backend.model;
 
```

Comment:
please make constructors with fields and from superclass in both combo and vendor